src/components/Box/Box.md

```diff
@@ -0,0 +1,32 @@
+### Notes for use
+
+A Box component can be used with any number of children and all of them will be inside the box.
+Use it in cases where you would consider using a Block but need a border.
+
+### Examples
+
+Basic usage:
+
+```js { "props": { "data-example": "basic" } }
+<Box>This is a Box</Box>
+```
+
+Box with MediaObject child:
+
+```js { "props": { "data-example": "Box with MediaObject child" } }
```

Comment:
data-example shouldn't have the component name, notice how "Box" is used twice in the screenshot's filename. should also be lowercased. "with-mediaobject-child" would be good.

Reply:
you'll have to manually delete this file before re-running tests after renaming this:
`config/styleguide-visual/reference/Box_Box_with_MediaObject_child_desktop.png`